akvo/rsr/tests/iati_checks/test_iati_checks_fields_countries_and_regions.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+"""
+Akvo RSR is covered by the GNU Affero General Public License.
+
+See more details in the license.txt file located at the root folder of the Akvo RSR module.
+For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+"""
+
+import datetime
+
+from akvo.rsr.tests.base import BaseTestCase
+from akvo.iati.checks.fields import countries_and_regions as geo_checks
+from akvo.rsr.models import RecipientCountry, RecipientRegion, Transaction
+
+
+class IatiCheckFieldsTransactionRecipientsTestCase(BaseTestCase):
+
+    def setUp(self):
+        self.project = self.create_project("Test project")
+        Transaction.objects.create(
+            project=self.project,
+            transaction_type="1",
+            transaction_date=datetime.date.today(),
+            value=1,
```

Comment:
This is just a random value, right? Not a constant?

Reply:
Here are the choices for `transaction_type` https://github.com/akvo/akvo-rsr/blob/add988b18841b16dfab92b8ce226d50e4d21f32d/akvo/codelists/store/codelists_v203.py#L5313